geonode_mapstore_client/client/js/plugins/VisualStyleEditor.jsx

```diff
@@ -123,15 +121,34 @@ function VisualStyleEditor({
     onCreate,
     onDelete,
     selectedStyle,
-    style: styleProp,
-    originalStyle,
-    onUpdateNode
+    style: styleProp
 }) {
 
     const [creating, setCreating] = useState(false);
     const [deleting, setDeleting] = useState(false);
     const [title, setTitle] = useState('');
 
+    // localstorage for style notification when cloning for the first time
+    const [styleNotification, setStyleNotification] = useLocalStorage('style-notifcation', {
+        style: layer?.availableStyles || [],
+        notificationOpen: false,
+        hasBeenClosed: false
+    });
```

Comment:
Thinking about it I would avoid to store all the styles inforamyion in the localStorage. I propose to change it with something like this

```
const [dismissStyleNotification, setDismissStyleNotification] = useLocalStorage('style-notifcation', false); // set true after clicking on 'Don't show this message again'
const [closeNotification, setCloseNotification] = useState(false);  // set true after clicking on th x button
```

the notification will be applied only if closeNotification and dismissStyleNotification are true

![image](https://user-images.githubusercontent.com/19175505/146161582-7756aaab-3c41-4262-be51-2e810d228155.png)